blog/blog.sql

```diff
@@ -2,7 +2,7 @@ CREATE TABLE blog
 (
     id serial PRIMARY KEY,
     user_name text,
```

Comment:
It might be a good idea to split out your users into another table, and have a foreign key from the `blog` table to the `users` table.